### PR TITLE
Fixed #31021 -- Fixed proxy model permissions data migration crash with a multiple databases setup.

### PR DESCRIPTION
**Pull request by felixxm**
ticket-31021

Regression in 98296f86b340c8c9c968375d59f1d3a3479e60c2.

### CODE REVIEW

**Comment by felixxm** [comment]
@haudoing Can you confirm that this fix works for you?

File: django/contrib/auth/migrations/0011_update_proxy_permissions.py [hunk 1]
```diff
@@ -39,7 +39,7 @@ def update_proxy_model_permissions(apps, schema_editor, reverse=False):
         old_content_type = proxy_content_type if reverse else concrete_content_type
         new_content_type = concrete_content_type if reverse else proxy_content_type
         try:
-            with transaction.atomic():
+            with transaction.atomic(using=schema_editor.connection.alias):
```
Comment (charettes) [inline django/contrib/auth/migrations/0011_update_proxy_permissions.py]: We likely want to be forcing the usage of `schema_editor.connection.alias` in the `ContentType` query above and the `Permission` one below as well else they could be against different database.

We also want to skip this function entirely if `not router.allow_migrate('auth', 'permission')`

Comment (felixxm) [reply django/contrib/auth/migrations/0011_update_proxy_permissions.py]: Thanks, updated :+1:

**Comment by haudoing** [comment]
@felixxm I've tested on your old one line fix and that works.

**Comment by felixxm** [comment]
@haudoing Thanks for checking :+1: The current solution should also work for you.

File: django/contrib/auth/migrations/0011_update_proxy_permissions.py (hunk 1 above)
Comment (felixxm) [reply django/contrib/auth/migrations/0011_update_proxy_permissions.py]: I added test for this case.

**Comment by haudoing** [comment]
Hi felixxm, 
Sorry if my question is too rude. But would you kindly let me know the estimated date of this fix get updated on pypi?

**Comment by felixxm** [comment]
Today🙂

**Pull request merged**